libs/api/domains/license-service/src/lib/mappers/huntingLicenseMapper.ts

```diff
@@ -145,6 +145,7 @@ export class HuntingLicensePayloadMapper implements GenericLicenseMapper {
               description: [
                 { text: formatMessage(m.huntingLicenseDescription) },
               ],
+              photo: t.holderPhoto,
```

Comment:
Why not` ?? undefined` here like in other places?

Reply:
I did it in the other place to coorce nulls into undefined, but the types match here. Don't think it matters at all.